warehouse/manage/views.py

```diff
@@ -1367,6 +1378,132 @@ def delete_organization(self):
         return HTTPSeeOther(self.request.route_path("manage.organizations"))
 
 
+@view_defaults(
+    route_name="manage.organization.projects",
+    context=Organization,
+    renderer="manage/organization/projects.html",
+    uses_session=True,
+    require_csrf=True,
+    require_methods=False,
+    permission="view:organization",
+    has_translations=True,
+    require_reauth=True,
+)
+class ManageOrganizationProjectsViews:
+    def __init__(self, organization, request):
+        self.organization = organization
+        self.request = request
+        self.user_service = request.find_service(IUserService, context=None)
+        self.organization_service = request.find_service(
+            IOrganizationService, context=None
+        )
+        self.project_factory = ProjectFactory(request)
+
+    @property
+    def active_projects(self):
+        return self.organization.projects
+
+    @property
+    def default_response(self):
+        active_projects = self.active_projects
+        all_user_projects = user_projects(self.request)
+        projects_owned = set(
+            project.name for project in all_user_projects["projects_owned"]
+        )
+        projects_sole_owned = set(
+            project.name for project in all_user_projects["projects_sole_owned"]
+        )
+        projects_requiring_2fa = set(
+            project.name for project in all_user_projects["projects_requiring_2fa"]
+        )
+        project_choices = set(
+            project.name
+            for project in all_user_projects["projects_owned"]
+            if not project.organizations
+        )
+        project_factory = self.project_factory
+
+        return {
+            "organization": self.organization,
+            "active_projects": active_projects,
+            "projects_owned": projects_owned,
+            "projects_sole_owned": projects_sole_owned,
+            "projects_requiring_2fa": projects_requiring_2fa,
+            "add_organization_project_form": AddOrganizationProjectForm(
+                self.request.POST,
+                project_choices=project_choices,
+                project_factory=project_factory,
+            ),
+        }
+
+    @view_config(request_method="GET")
+    def manage_organization_projects(self):
+        if self.request.flags.enabled(AdminFlagValue.DISABLE_ORGANIZATIONS):
+            raise HTTPNotFound
+
+        return self.default_response
+
+    @view_config(request_method="POST")
```

Comment:
Seems the `view_config` here should only be accessible to those with `manage:organization` permissions on the org. Currently the class level permission is `view:organization`, so any member could add a project?

Reply:
Fixed with commit [`6f4928a`](https://github.com/pypa/warehouse/pull/11473/commits/6f4928a129bbb81acea7b0ea9710343364a3b4eb).